src/pages/components/modal/style.mdx

```diff
@@ -193,3 +210,31 @@ the content while sticking to the bottom of the mobile screen.
 
 </Column>
 </Row>
+
+## AI presence
+
+The following are the unique styles applied to the component when the AI slug is
+present. Unless specified, all other tokens in the component remain the same as
+the non-AI variant.
+
+More information about AI style elements is coming soon.
+
+| Element                    | Property         | Token / Size        |
+| -------------------------- | ---------------- | ------------------- |
+| Modal:background           | background-color | `layer-01`          |
+|                            | box-shadow       | `$ai-drop-shadow`   |
+|                            | inner-shadow     | `$ai-inner-shadow`  |
+| Overlay                    | background-color | `$ai-overlay`       |
+| Linear gradient:background | start            | `$ai-aura-start-sm` |
+|                            | stop             | `$ai-aura-stop`     |
+| Linear gradient:border     | top              | `$ai-border-start`  |
+|                            | bottom           | `$ai-border-end`    |
+| AI Slug                    | size             | large               |
+
+<Row>
+<Column colLg={8}>
+
+![Modal AI presence](images/modal-usage-style-presence.png)
```

Comment:
There's a typo in the image with punctuation: "however, "
<img width="440" alt="image" src="https://github.com/carbon-design-system/carbon-website/assets/11670886/217e72e1-fab1-475b-b77d-f30db6b98da4">